pkg/scene/scan.go

```diff
@@ -333,3 +357,48 @@ func getInteractive(path string) bool {
 	_, err := os.Stat(GetFunscriptPath(path))
 	return err == nil
 }
+
+func getCaption(path string) string {
+	captions := ""
+	parent := filepath.Dir(path)
+	baseName := filepath.Base(path)
+
+	// only proceed if we haven't already looked at this directory
+	if previousDir != parent {
+		files, err := ioutil.ReadDir(parent)
+		if err != nil {
+			return captions
+		}
+		// if directory contains more file than specified value assume no organization
+		if len(files) < 20 {
+			videoExt := filepath.Ext(path)
+			videoName := strings.TrimSuffix(baseName, videoExt)
+			for _, file := range files {
+				filename := file.Name()
+				fileExt := filepath.Ext(filename)
+				if fileExt == ".vtt" || fileExt == ".srt" {
+					vttName := strings.TrimSuffix(filename, fileExt)
+					languageCode := filepath.Ext(vttName)
+					if vttName == videoName {
+						languageCode = "?"
```

Comment:
The language code string will be returned back as a url query path. I think we should avoid using a url query reserved char that we might need to urlescape/encode at some point

Reply:
Good point.